batteries/pp.luau

```diff
@@ -21,20 +21,28 @@ local function isPrimitiveArray(array: { [unknown]: unknown })
 	return n == max
 end
 
-local function formatValue(value: unknown)
-	if typeof(value) ~= "string" then
+local function formatValue(value: unknown, options: { [string]: unknown }?)
```

Comment:
you should define an exported type `PrettyPrintOptions` and it should have the actual list of options currently supported, and that should be the type of this, not `{[string]: unknown}`.